### PR TITLE
@damassi => Support collecting institution works

### PR DESCRIPTION
**Pull request by mzikherman**
Pretty simple, we don't display any partner or commercial info, except to display the value of this field.

Looks like:

screen shot 2018-12-07 at 5 02 17 pm

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 49a0b1c171e9d8c328405e506e73c2d047e16c5e

https://deploy-preview-1655--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 261e5e7b1cec917faa951487e7ba14667be7a58b

https://deploy-preview-1655--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx [hunk 1]
```diff
@@ -39,6 +39,10 @@ export class ArtworkSidebarPartnerInfo extends React.Component<
 
   render() {
     const { artwork } = this.props
+    const { collecting_institution } = artwork
```
Comment (oxaudo) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx]: Oh @mzikherman - I think the latest on that was to display partner info for those works and not the collecting institution field. I believe that was intentional departure from what is currently displayed on artwork page.

There were pretty long discussions about that and I think the idea is that we want to encourage to correctly convert this 'collecting institution' field to proper availability.

Following this discussion this collecting_institution field was removed:
https://github.com/artsy/reaction/pull/1565

Wish we could tag Steven here but not sure about his github login:)

Comment (mzikherman) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx]: Doh! I wish that were reflected in the comps I was going off of :(

Comment (oxaudo) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx]: Yea, those keep changing and not always one to one with the latest accepted decisions. It is confusing though. Might be worth bringing at retro that when those things change - at least slap something like 'outdated' to the design spec.

**Pull request merged**